src/Umbraco.Core/Models/ContentEditing/Language.cs

```diff
@@ -13,7 +13,8 @@ public class Language
         [Required(AllowEmptyStrings = false)]
         public string? IsoCode { get; set; }
 
-        [DataMember(Name = "name")]
+        [DataMember(Name = "name", IsRequired = true)]
+        [Required(AllowEmptyStrings = false)]
         public string? Name { get; set; }
```

Comment:
Should this not be `public string Name ... ` then, instead of nullable?

Reply:
That would make more sense yes (same with `IsoCode`), although it's technically valid to set a `null` value: it just won't pass model validation 🤔

Reply:
I've changed this for the `IsoCode` property and made `Name` optional, falling back to the English culture name retrieved from the ISO code (within the `LanguageController`👍🏻